app/controllers/v0/zipcode_rates_controller.rb

```diff
@@ -2,10 +2,11 @@
 
 module V0
   class ZipcodeRatesController < ApiController
-    skip_before_action :resolve_version
-    # GET /v0/housing_rates/20001
+    # GET /v0/zipcode_rates/20001
     def show
-      resource = ZipcodeRate.find_by(zip_code: params[:id])
+      resource = ZipcodeRate.version(@version[:number])
```

Comment:
It would be more performant if you wrote it as:  then you could use a composite index. 
`resource = ZipcodeRate.where(version: @version[:number], zip_code: params[:id]).order(:mha_rate).first`

Reply:
I take that back.  They both end up with the same sql, though I do think the suggested version is clearer to read.

Reply:
Can do!